_rules/link-accessible-name-c487ae.md

```diff
@@ -69,72 +70,72 @@ For `area` elements that have a `href` attribute, but are not nested inside a `m
 
 #### Passed Example 1
```

Comment:
Since we're updating the examples anyway, can you make sure the accessible name is appropriate for the link and the image?